templates/index.html

```diff
@@ -1,10 +1,44 @@
 <!doctype html>
-  <title>Upload New Image</title>
-  <h1>Upload New Image</h1>
-  <form method=post enctype=multipart/form-data>
-    <input type=file name=file>
-    <input type=submit value=Upload>
-  </form>
-  <img src="{{ uploadedImagePath }}" alt="Uploaded Image" width="800px">
-  <h2>Results</h2>
-  <p>nothing yet</p>
+<title>Upload New Image</title>
+<head>
+  <link rel="stylesheet" href="{{ url_for('static', filename='css/scroll.css') }}">
+</head>
+<body>
+  <header>
+    <h1>What's My Style?</h1>
+  </header>
+  <div class="panel" id="options">
+    <h2>Settings</h2>
+    <p>What's my:</p>
+    <form>
+      <input type="checkbox" id="style" name="style" value="style">
+      <label for="style"> Style</label><br>
+      <input type="checkbox" id="genre" name="genre" value="genre">
+      <label for="genre"> Genre</label><br>
+      <input type="checkbox" id="artist" name="artist" value="artist">
+      <label for="artist"> Artist</label><br><br>
+    </form>
```

Comment:
I noticed earlier during the demo recording that all of the options were centre justified. Can you make them left justified so that the checkboxes all line up?